tests/CMakeLists.txt

```diff
@@ -53,6 +53,21 @@ set(tests
         test_sockopt_hwm
         test_heartbeats
         test_poller
+        test_atomics
+        test_bind_src_address
+        test_capabilities
+        test_ipc_wildcard
```

Comment:
That can be arranged.
Please review all the tests in `tests/CMakeLists.txt`.
Might it be possible that a test is only included in the Linux section while the feature-under-test has a Windows implementation?